client/src/components/feedback/comment.js

```diff
@@ -14,7 +14,7 @@ class FeedbackComment extends Component {
           <small className="date">
             {
               // eslint-disable-next-line max-len
```

Comment:
You can use `+` operator to concatenate two template strings

Reply:
Due to tradition, when I try this - it doesn't work. But when you give an advice - it works! Thanks!